scheduler/src/cook/components.clj

```diff
@@ -436,7 +434,20 @@
                      (when enabled?
                        (when (zero? port)
                          (throw (ex-info "You enabled nrepl but didn't configure a port. Please configure a port in your config file." {})))
-                       ((lazy-load-var 'clojure.tools.nrepl.server/start-server) :port port)))}))
+                       ((lazy-load-var 'clojure.tools.nrepl.server/start-server) :port port)))
+     :retrieve-url-path-fn (fnk [[:config {agent-query-cache {:threshold 1000, :ttl 6000}}]]
```

Comment:
Should we add documentation of this new configurability? Also, should we add an entry in the changelog?